Pod/Classes/TTRangeSlider.m

```diff
@@ -647,6 +647,21 @@ -(void)setLabelPadding:(CGFloat)labelPadding {
     [self updateLabelPositions];
 }
 
+-(void)setEnableHandleShadow:(BOOL)enableHandleShadow {
+    _enableHandleShadow = enableHandleShadow;
+    if (enableHandleShadow) {
+        self.leftHandle.shadowRadius = 2;
+        self.leftHandle.shadowOffset = CGSizeMake(-1,1);
+        self.leftHandle.shadowColor = [UIColor grayColor].CGColor;
+        self.leftHandle.shadowOpacity = 0.8;
+        
+        self.rightHandle.shadowRadius = 2;
+        self.rightHandle.shadowOffset = CGSizeMake(1,1);
+        self.rightHandle.shadowColor = [UIColor grayColor].CGColor;
+        self.rightHandle.shadowOpacity = 0.8;
+    }
```

Comment:
how about turning shadows off?
if not, then just convert this to a function `- (void) enableHandleShadow {`